### ISSUE TITLE
论文和代码中的疑惑

### ISSUE THREAD

**Issue by so-bright** [murufeng/EPSANet#14]
您好，论文中的图示和代码实现的不一致，是论文里图画错了吗

**Comment by zhululmt** [comment]
您好，Is out = torch.cat((out, x_se_weight_fp), dim=1)?   是否写反了

![image](https://user-images.githubusercontent.com/48901525/146004147-84adec52-15d7-4bab-a226-120916887d94.png)

**Comment by ndsclark** [comment]
> 您好，Is out = torch.cat((out, x_se_weight_fp), dim=1)? 是否写反了
> 
> ![image](https://user-images.githubusercontent.com/48901525/146004147-84adec52-15d7-4bab-a226-120916887d94.png)

我也认为应该是写反了
